orttraining/orttraining/training_ops/cpu/controlflow/ort_tasks.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/ml_value.h"
+
+#include <mutex>
+#include <future>
+
+
+namespace onnxruntime {
+namespace contrib {
+
+typedef std::pair<Status, std::vector<OrtValue>> ForwardReturnType;
+typedef std::pair<bool, std::vector<OrtValue>> BackwardReturnType;
+
+class OrtTasks final {
+ public:
+  static OrtTasks& GetInstance() {
+    static OrtTasks* instance_ = new OrtTasks;
```

Comment:
>static OrtTasks* instance_ = new OrtTasks; [](start = 4, length = 42)

this can be `static OrtTasks instance;`

Reply:
this was originally implemented as `static OrtTasks instance;`, but suggested to use raw pointer by Derek. 

See original comments here: